west.yml: Update hal_silabs to pull request #53 | Update to GSDK 4.4.0

sateeshkotapati opened the pull request:
Updated the files present in gecko/se_manager folder in Silabs HAL as per latest version of gecko_sdk.

This pull request is required to align the codebase of zephyr_rtos/hal_silabs (https://github.com/zephyrproject-rtos/hal_silabs) with gecko_sdk v4.4.0 (https://github.com/SiliconLabs/gecko_sdk)

PR in hal_silabs : https://github.com/zephyrproject-rtos/hal_silabs/pull/53

sateeshkotapati committed:
gecko: se_manager files updated | Update to GSDK 4.4.0

Updated the files present in gecko/se_manager folder in Silabs HAL
as per latest version of gecko_sdk.

Signed-off-by: Sateesh Kotapati <sateesh.kotapati@silabs.com>

sateeshkotapati commented:
Hi @fkokosinski! could you have a look this PR, Thanks.

fkokosinski commented:
Hi @sateeshkotapati, thanks for the PR. I've merged the HAL PR. You can now bump the HAL revision to `11ab59175a5ded618680a7692dbaae8f4fbd6325`.

sateeshkotapati commented:
> You can now bump the HAL revision to `11ab59175a5ded618680a7692dbaae8f4fbd6325`.

Hi @fkokosinski, I have rebased and changed the revision, please check and approve the PR, Thanks.
